website/docs/sql_ddl.md

```diff
@@ -40,6 +40,13 @@ CREATE TABLE IF NOT EXISTS hudi_table (
 ) USING hudi;
 ```
 
+:::note NOTE:
```

Comment:
This doesn't seem specific to non-partitioned table. Can we move this to be under `Create table` section instead of here. May be move it  to line 29 - https://github.com/apache/hudi/pull/10201/files#diff-7a654d7c7dc6889175056d700c94e58990c95944f8750e9a254bfedb3201ecd1R29 ?

Reply:
done